Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -317,9 +505,29 @@ def main() -> None:
             )
             demisto.setLastRun(next_run)
             demisto.incidents(incidents)
+        elif command == "reco-update-incident-timeline":
+            reco_client.update_reco_incident_timeline(
+                incident_id=demisto.args()["incident_id"],
+                comment=demisto.args()["comment"],
+            )
+            return_results("Incident timeline updated successfully")
```

Comment:
the return_results should get a `CommandResults` object
So maybe change to something like :
return_results(CommandResults(readable_output="Incident timeline updated successfully"))
Also worth adding the raw response as discussed in the demo

Reply:
Done